lib/desired-caps.js

```diff
@@ -120,6 +119,12 @@ const desiredCapConstraints = {
   ignoreAboutBlankUrl: {
     isBoolean: true
   },
+  startIWDP: {
+    isBoolean: true
+  },
+  iwdpLogging: {
```

Comment:
I'd rathe rename it to `showIWDPLog`, so it's consistent with other similar caps